model/jvm/src/test/scala/explore/model/CirceGoldenSuite.scala

```diff
@@ -9,5 +9,5 @@ import io.circe.testing.ArbitraryInstances
 import munit.DisciplineSuite
 
 class CirceGoldenSuite extends DisciplineSuite with ArbitraryInstances {
-  checkAll("GoldenCodec[ObsSummary]", GoldenCodecTests[ObsSummary](5).goldenCodec)
+  checkAll("GoldenCodec[ObsSummary]", GoldenCodecTests[ObsSummary](10).goldenCodec)
```

Comment:
Great to see this getting traction. Why did you increase the amount of samples? were there conditions not included?

Reply:
I was trying to get a mix of `pointings` and contraint sets. It was hard to get all of the permutations. I guess another option would be to set the number high when generating samples, then delete some of the generated cases with overlap and set the number to match how many files are left.